File: JavaToPython/java_to_python7.py
# ChristianThompson.com
# @TokyoEdTech
# Lesson 7: Lists

scores = []

scores.append(85)
scores.append(90)
scores.append(100)

print()
for i in range(0, len(scores)):
    print("Score {}: {}".format(i, scores[i]))

average_score = (scores[0] + scores[1] + scores[2]) / 3.0
print("Average Score: {}".format(average_score))

heights = [150, 160, 170, 180, 190]

print()
for i in range(0, len(heights)):
    print("Height {}: {}".format(i, heights[i]))

for height in heights:
    print(height)

total = 0
for height in heights:
    total += height
average_height = total/len(heights)
print("Average Height: {}".format(average_height))

stuff = [0, 2.2, "Bob", True]
for thing in stuff:
    print(thing)

File: JavaToPython/JavaToPython7.java
// ChristianThompson.com
// @TokyoEdTech
// Lesson 7: Arrays

class Main {
  public static void main(String[] args) {
    int[] scores = new int[3];

    scores[0] = 85;
    scores[1] = 90;
    scores[2] = 100;

    System.out.println("");
    for(int i=0; i<scores.length; i++){
      System.out.println("Score "+i+": "+ scores[i]);
    }

    double averageScore = (scores[0] + scores[1] + scores[2]) / 3.0;
    System.out.println("Average Score: " + averageScore);
    
    int heights[] = {150, 160, 170, 180};
    
    System.out.println("");
    for(int i=0; i<heights.length; i++){
      System.out.println("Height "+i+": "+ heights[i]);
    }
    
    System.out.println("");
    for(int height: heights){
        System.out.println(height);
    }
    
    System.out.println("");
    double averageHeight = (heights[0] + heights[1] + heights[2] + heights[3]) / 4.0;
    System.out.println("Average Height: " + averageHeight);

    

  }
}
